lib/matplotlib/rcsetup.py

```diff
@@ -501,6 +501,8 @@ def __call__(self, s):
 
 # a map from key -> value, converter
 defaultParams = {
+    'style': ['', validate_stringlist],
```

Comment:
Personally, I see two broad categories for style sheets: Those focused on aesthetics and those focused on layout (for slides, journal papers, etc.). I think these should be combined at runtime with `style.use`. I don't see the need for more fine grained partitioning, but I imagine others would. I don't want to push too much, but I think a simpler initial implementation is a good idea.